src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborReaderTests.Integer.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    public partial class CborReaderTests
+    {
+        // Data points taken from https://tools.ietf.org/html/rfc7049#appendix-A
+
+        [Theory]
+        [InlineData(0, "00")]
```

Comment:
I'd love to see a fuzz tester that looks over all `[InlineData(...)]` attributes and mutates over that for all of our data format parsers.

Reply:
We use [MSRD](https://www.microsoft.com/en-us/security-risk-detection/) for fuzzing parsers. But the scenario you describe [can also be accommodated using FsCheck](https://github.com/eiriktsarpalis/runtime/blob/8d799242a79e6cce044e5a0738581fd55784f8a0/src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborRoundtripTests.cs).